pyproject.toml

```diff
@@ -59,16 +59,15 @@ as-redis-signal = ["as-redis-signal"]
 
 
 [tool.poetry.dev-dependencies]
-black = "22.3.0"
-pre-commit = "2.18.1"
-pytest = "7.1.1"
+black = "^22.3.0"
+pre-commit = "^2.18.1"
+pytest = "^7.1.1"
 alabaster = "0.7.12"
-pylint = "2.13.4"
-flake8 = "4.0.1"
-isort = "5.10.1"
+pylint = "^2.13.4"
+flake8 = "^4.0.1"
+isort = "^5.10.1"
```

Comment:
PyLint, Black, Flake8 and Isort used to be required dependencies, in order to use these tools in VSCode, but now VSCode packages these tools with the individual extensions, so we don't really need to specify them as dev dependencies anymore.

> A Visual Studio Code extension with support for the Pylint linter. This extension ships with pylint=3.0.3.

https://marketplace.visualstudio.com/items?itemName=ms-python.pylint#pylint-extension-for-visual-studio-code